library/alloc/src/vec.rs

```diff
@@ -159,7 +159,7 @@ use crate::raw_vec::RawVec;
 /// # Slicing
 ///
 /// A `Vec` can be mutable. Slices, on the other hand, are read-only objects.
-/// To get a slice, use `&`. Example:
+/// To get a slice, use [`&`]. Example:
```

Comment:
This is a broken link. If you want it to link to references, you need to use a relative link (not an intra-doc link: https://github.com/rust-lang/rust/issues/63351).

Reply:
I added a link.